src/plugins/data/common/search/es_search/types.ts

```diff
@@ -16,14 +16,15 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { SearchParams, SearchResponse } from 'elasticsearch';
+import { SearchResponse } from 'elasticsearch';
```

Comment:
Should this be imported from the legacy package? If so, it might be nice to rename this import to make it clearer:

```suggestion
import { SearchResponse as LegacySearchResponse } from 'elasticsearch';
```

Reply:
Removed